velox/common/base/Status.h

```diff
@@ -156,6 +156,8 @@ class [[nodiscard]] Status {
     }
   }
 
+  explicit Status(StatusCode code);
```

Comment:
would it not want to allow implicit conversions here? I guess if you remove the `explicit` you could do things like returning the error code directly in a function supposed to return Status, which could be convenient.

Reply:
Hmm... I'm just used to add 'explicit' to single-arg constructors. Otherwise, linter complains.

Here, the difference would be `return Status::UserError()' vs. 'return StatusCode::kUserError'. The letter is longer, so not sure if there is a benefit in allowing that.

Reply:
I don't feel strongly about it, but for example if you already have the status code it might be more convenient to do 

```
return code;
```

rather than

```
return Status(code);
```

and to get rid of the lint you probably need to explicitly add an "/* implicit */" comment to the constructor.